<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VultusMainWindow</class>
 <widget class="QMainWindow" name="VultusMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VultusMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #DCDCDC;</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>950</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(245, 245, 245,80%);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="employees_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift SemiBold</family>
         <pointsize>12</pointsize>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(245, 245, 245,0%);
border: 0;</string>
       </property>
       <property name="text">
        <string>СОТРУДНИКИ</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="profile_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift SemiBold</family>
         <pointsize>12</pointsize>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(245, 245, 245,0%);
border: 0;</string>
       </property>
       <property name="text">
        <string>ПРОФИЛЬ</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="tasks_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift SemiBold</family>
         <pointsize>12</pointsize>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(245, 245, 245,0%);
border: 0;</string>
       </property>
       <property name="text">
        <string>ЗАДАЧИ</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>400</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="main_place">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>950</width>
      <height>690</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="employees_page">
     <widget class="QLineEdit" name="search_line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>391</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(245, 245, 245,50%); 
border: 0;
border-radius: 14px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="search_icon_label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>28</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(245, 245, 245,0%);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QScrollArea" name="profile_scroll_area">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>390</width>
        <height>630</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="profile_layout_area">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>390</width>
         <height>630</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>210</width>
        <height>270</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(245, 245, 245,80%);</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>310</width>
        <height>270</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(245, 245, 245,80%);</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>290</y>
        <width>530</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(245, 245, 245,80%);</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>360</y>
        <width>530</width>
        <height>320</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(245, 245, 245,80%);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="profile_page">
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>230</width>
        <height>310</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(245, 245, 245,80%);</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>690</width>
        <height>310</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(245, 245, 245,80%);</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_8" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>930</width>
        <height>350</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(245, 245, 245,80%);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tasks_page">
     <widget class="QWidget" name="widget_9" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>930</width>
        <height>310</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(245, 245, 245,80%); border: 0;</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_10" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>930</width>
        <height>350</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(245, 245, 245,80%); border: 0;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
